ososedki_dl/crawlers/other/fapello_is.py

```
@@ -37,10 +37,19 @@ async def fetch_media_urls(
             response is not a list.
         """
         headers: dict[str, str] = {"Referer": referer_url}
-        async with self.session.get(url, headers=headers) as response:
-            if response.status != 200:
-                return []
-            return await response.json()
+        try:
+            return await self.downloader.fetch(
+                url, headers=headers, response_property="json"
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)